src/renderer/Renderer.js

```diff
@@ -51,6 +52,47 @@ export default class Renderer {
         this.dataframes = [];
     }
 
+    /**
+     * Initialize renderer:
+     * - WebGL context
+     * - Map zoom and center
+     * @param {Map} map
+     * @param {WebGLRenderingContext} gl
+     */
+    initialize (map, gl) {
```

Comment:
Passing the map here seems like a bad idea to me since this is coupling the renderer with the map

Reply:
Yes, you are right